api/v1beta1/network_disruption_test.go

```diff
@@ -270,6 +283,29 @@ var _ = Describe("NetworkDisruptionSpec", func() {
 			Entry("custom path and method", "/test", "delete"),
 		)
 	})
+	When("NetworkDisruptionServiceSpecFromString is called", func() {
+		It("handles ports with non-alpha names", func() {
+			expected := []NetworkDisruptionServiceSpec{{
+				Name:      "demo-service",
+				Namespace: "demo-namespace",
+				Ports: []NetworkDisruptionServicePortSpec{
+					{
+						Name: "demo-port",
+						Port: 8080,
+					},
+					{
+						Port: 8180,
+					},
+				},
+			}}
+
+			testString := []string{"demo-service;demo-namespace;8080-demo-port;8180-"}
+
+			actual, err := NetworkDisruptionServiceSpecFromString(testString)
+			Expect(err).ShouldNot(HaveOccurred())
+			Expect(actual).Should(Equal(expected))
```

Comment:
Should we check if the error message contains `service port format is expected to follow...` and/or `port format is expected to be a valid integer...`?

Reply:
Fax! It is just a suggestion, nothing required. Usually I test error messages when my function can return multiple errors.